projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/TransformJob.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { jobData } from '../jobData';
+import { VdkOption } from '../vdkOptions/vdk_options';
+import VDKTextInput from './VdkTextInput';
+import { Dialog, showDialog } from '@jupyterlab/apputils';
+import { jobTransformRequest } from '../serverRequests';
+import { IJobPathProp } from './props';
+import { VdkErrorMessage } from './VdkErrorMessage';
+
+export default class TransformJobDialog extends Component<IJobPathProp> {
+  /**
+   * Returns a React component for rendering a transform menu.
+   *
+   * @param props - component properties
+   * @returns React component
+   */
+  constructor(props: IJobPathProp) {
+    super(props);
+  }
+  /**
+   * Renders a dialog for transforming a data job.
+   *
+   * @returns React element
+   */
+  render(): React.ReactElement {
+    return (
+      <>
+        <VDKTextInput
+          option={VdkOption.NAME}
+          value="default-name"
+          label="Job Name:"
```

Comment:
We are doing transform job only to locally installed jobs. So the user should firstly download it  and then when he has it, he should do the transform in this way I think we can only ask for the directory

Reply:
Makes sense, yeah, Will remove the job name and team name.